### PR TITLE
#302 Cors.Permissive() add Authorization header

### PR DESCRIPTION
**Pull request by wcontayon**
#302 
Authorization header in Access-Control-Allow-Headers

### CODE REVIEW

File: Testing/Modules/Security/CorsTests.cs [hunk 1]
```diff
@@ -42,6 +42,28 @@ public async Task TestPermissive()
 
             Assert.AreEqual("*", response.GetHeader("Access-Control-Allow-Origin"));
 
+            Assert.AreEqual("*", response.GetHeader("Access-Control-Allow-Methods"));
+            Assert.AreEqual("Authorization", response.GetHeader("Access-Control-Allow-Headers"));
```
Comment (Kaliumhexacyanoferrat) [inline Testing/Modules/Security/CorsTests.cs:46]: I think, this should be `*, Authorization`

File: Modules/Security/CorsPolicy.cs [hunk 2]
```diff
@@ -10,7 +10,15 @@ public static class CorsPolicy
         /// Creates a policy that does not restrict browsers from interacting
         /// with the requested resources.
         /// </summary>
-        public static CorsPolicyBuilder Permissive() => new CorsPolicyBuilder().Default(new OriginPolicy(null, null, null, true, 86400));
+        /// <param name="defaultAuthorizationHeader">Indicate if the header Authorization should be in 'Access-Control-Allow-Headers'. Default value=false</param>
```
Comment (Kaliumhexacyanoferrat) [inline Modules/Security/CorsPolicy.cs:13]: "Default value=false", but default value is set to `true`?

Comment (Kaliumhexacyanoferrat) [reply Modules/Security/CorsPolicy.cs:13]: Regarding the naming, probably something like `allowAuthorization` would express the intention more clearly?

**Review by Kaliumhexacyanoferrat — CHANGES_REQUESTED** [review]
Thanks for the PR, I left a few comments to look over.

File: Testing/Modules/Security/CorsTests.cs (hunk 1 above)
Comment (wcontayon) [reply Testing/Modules/Security/CorsTests.cs]: Yep, I've fixed it 🙂

File: Modules/Security/CorsPolicy.cs (hunk 2 above)
Comment (wcontayon) [reply Modules/Security/CorsPolicy.cs]: It was a comment mistake. Fixed !!. The variable has been changed => allowAuthorization

**Review by wcontayon — COMMENTED** [review]
Change applied

**Review by Kaliumhexacyanoferrat — APPROVED** [review]

**Comment by Kaliumhexacyanoferrat** [comment]
Perfect, thanks 👍

**Pull request merged**